docs/deployment-guide/goals/production/README.md

```diff
@@ -79,8 +79,8 @@ The respective compatible versions can be found in the [CHANGELOG.md](../../../.
 - The domain under which the Keycloak should be reachable on the internet will be referred to as `[KC_FQDN]` in this
   guide and should differ from the `[AP_FQDN]`.
 - The steps to set up the realm are the following
-   1. Copy [mds-theme](../../../../authority-portal-keycloak/mds-theme) directory to `{keycloakRoot}/themes/` directory
-   2. Import [realm.json](../../../../authority-portal-backend/authority-portal-quarkus/src/main/resources/realm.json) to create `mds-portal` realm
+   1. Copy [mds-theme](../../../../authority-portal-keycloak/mds-theme) or [sovity-theme](../../../../authority-portal-keycloak/sovity-theme) directory to `{keycloakRoot}/themes/` directory
+   2. Import [realm-mds.json](../../../../authority-portal-backend/authority-portal-quarkus/src/main/resources/realm-mds.json) to create `mds-portal` realm or [realm.json](../../../../authority-portal-backend/authority-portal-quarkus/src/main/resources/realm.json) to create `authority-portal` realm
```

Comment:
Since there are two paths of either setting up the sovity AP or MDS AP we could just give instructions for the separate cases for more clarity aka

Case sovity Portal:
1. Copy ....
2. Import ...

Case MDS Portal:
1. Copy...
2. Import ...


can use this for the docs in general